components/o3-editorial-typography/src/tsx/detail.tsx

```diff
@@ -1,11 +1,20 @@
-import {DetailProps, TopicTagProps, QuoteProps, BigNumberProps, BylineProps} from '../types';
+import {
+	DetailProps,
+	TopicTagProps,
+	QuoteProps,
+	BigNumberProps,
+	BylineProps,
+} from '../types';
 import {getDataAttributes} from './utils';
 
 export const TopicTag = ({href, theme, children}: TopicTagProps) => {
 	const attributes = getDataAttributes(theme);
 	const HtmlElement = href ? 'a' : 'span';
 	return (
-		<HtmlElement href={href} className="o3-editorial-typography-topic-tag" {...attributes}>
+		<HtmlElement
+			href={href}
+			className="o3-editorial-typography-topic-tag"
+			{...attributes}>
```

Comment:
The `href` attribute will still be present when `HtmlElement` is a `span`. A cleaner approach would be to use the ternary to change what element is returned.
```suggestion
return href ? <a href className="o3-editorial-typography-topic-tag">{children}</a> : <span className="o3-editorial-typography-topic-tag">{children}</span>
```